server/models/ConnectedAccount.js

```diff
@@ -72,5 +72,9 @@ export default (Sequelize, DataTypes) => {
     });
   };
 
+  ConnectedAccount.prototype.delete = async function() {
+    return this.destroy();
+  };
+
```

Comment:
No need to add more code, we can use the `destroy` function directly

Reply:
resolved with https://github.com/opencollective/opencollective-api/pull/2672/commits/6e3ad30b03e212430eb30a015eb716703ec39d96